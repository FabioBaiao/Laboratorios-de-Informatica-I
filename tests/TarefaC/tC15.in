aAA
cBa
0 1 E
ALALDADSL