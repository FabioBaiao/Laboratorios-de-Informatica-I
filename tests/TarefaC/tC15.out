1 1
2 1
1 0
FIM 9
